### ISSUE TITLE
No warning is given and the platform is not added when min and max versions are left as "---"

### ISSUE THREAD

**Issue by ioanarusiczki** [mozilla/addons-server#11121]
First scenario:
1. Login dev hub and submit an add-on, make it available only for one platform (FF or Android).
2. After the submission is completed go to Manage version page and add the other platform.
3. Don't change the min and max versions, leave them as it is "---"
4. Click "Save changes" and observe the screen 

Second scenario:
1. Login dev hub and go to Manage version page of an addon that has FF and Android platforms available
2. Remove one of the platforms and click the "Save changes" button
3. After changes are successfully saved add back the previous removed platform
4. Click "Save changes" again

Expected result:
A red message is displayed warning the user that a min and max version is required.
Actual result:
"Changes successfully saved" appears but the platforms remains removed.

Notes:
Issue is reproducible on all AMO servers - FF66(Win10) for listed and unlisted add-ons which means that it's not caused by the django 2.2 upgrade https://github.com/mozilla/addons-server/issues/11020

![no changes to add back the platform](https://user-images.githubusercontent.com/33448286/55475854-b0195000-561d-11e9-90c4-66a9cd81aa6c.gif)

**Comment by diox** [comment]
Not a huge deal as long as it works when you select versions, but I agree it's not great, we should validate the versions and return an error message if necessary.

**Comment by Patil2099** [comment]
@diox  do we have to add a validation to the forms.py file. If yes, Can I take this?

**Comment by diox** [comment]
Yes. You might run into other issues though, since we display the compatible apps dynamically. Try it out locally!

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDSRV-10
